localstack/testing/pytest/fixtures.py

```diff
@@ -1441,6 +1458,25 @@ def _create_role_and_policy(**kwargs):
         iam_client.delete_role(RoleName=role_name)
 
 
+@pytest.fixture
+def firehose_create_delivery_stream(firehose_client, wait_for_delivery_stream_ready):
+    delivery_streams = {}
+
+    def _create_delivery_stream(**kwargs):
+        if "DeliveryStreamName" in kwargs:
```

Comment:
nit: should it be `not in`?

Reply:
@bentsku Thanks for catching this, I will be more careful.

Reply:
No worries, that's what PR reviews are for 😁